Source/NaturalLanguageClassifierV1/Models/ClassifierModel.swift

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright IBM Corporation 2015
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Freddy
+
+extension NaturalLanguageClassifierV1 {
+    
+    public struct ClassifierModel: JSONDecodable {
```

Comment:
I like how you named the models. It's unfortunate that the service has so much overlap between this model and the previous one. But I like your use of `ClassifierModel` and `ClassifierDetails` to disambiguate the two.